paper-toast.html

```diff
@@ -48,6 +48,7 @@
 ----------------|-------------|----------
 `--paper-toast-background-color` | The paper-toast background-color | `#323232`
 `--paper-toast-color` | The paper-toast color | `#f1f1f1`
+`--paper-font-common-base` | Mixin applied to the toast for setting the font, defined in `paper-styles/typography.html` | `{}`
```

Comment:
I think the description needs to be a bit better. How about: Mixin applied to the toast for setting the font. If you want to use the Roboto font, you can just import `paper-styles/typography.html` when using this element; otherwise, you can define your own font style. WDYT?